index.html

```diff
@@ -5,13 +5,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
-    <link href="./main_style.css" type="text/css" rel="stylesheet" />
-    <script src="https://kit.fontawesome.com/6a5b34451f.js" crossorigin="anonymous"></script>
+    <link href="styles/main_styles.css" type="text/css" rel="stylesheet" />
+    <script
+      src="https://kit.fontawesome.com/6a5b34451f.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
   <body>
     <header>
       <div class="logo">My logo</div>
```

Comment:
- [ ] [REQUIRED] Kindly add a link to the logo `a` tag as stated in the requirement